testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-spark-hive/spark-defaults.conf

```diff
@@ -7,3 +7,5 @@ spark.hive.metastore.uris=thrift://hadoop-master:9083
 spark.hive.metastore.warehouse.dir=hdfs://hadoop-master:9000/user/hive/warehouse
 spark.hive.metastore.schema.verification=false
 spark.hive.enforce.bucketing=false
+# TODO: Enable vectorized reader once Parquet V1 files can be written (https://github.com/trinodb/trino/issues/7953)
+spark.sql.parquet.enableVectorizedReader=false
```

Comment:
https://github.com/trinodb/trino/issues/7953

Reply:
Can `spark.sql.parquet.enableVectorizedReader=false` be set not globally, but within particular test method. We should still test current default Parquet writer's with Spark's default setup.

Since we want to test on defaults as much as possible, for native Parquet writer we should have both tests: with Spark vectorized reader off and on.

Please add this test coverage as a prep commit. Then, in the "Add created_by field" it will be apparent how this changes the tested behavior.

Reply:
I can set the session property in the method body, but the vectorized reader does not work currently. I'd rather focus on closing this issue out and just fixing that next.